### PR TITLE
[quant][graphmode][fx] Support input_quantized_idxs and output_quantized_idxs in the new convert

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #68229
* #68228
* #68227
* __->__ #68042
* #67537
* #68602

Summary:
att

Also added test cases from TestQuantizeFx which tests all combinations of {fp32, int8} input and output override

Test Plan:
```
python test/fx2trt/test_quant_trt.py TestConvertFxDoNotUse
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D32271511](https://our.internmc.facebook.com/intern/diff/D32271511)

### CODE REVIEW

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32271511).

File: torch/ao/quantization/fx/_convert_do_not_use.py [hunk 1]
```diff
@@ -182,16 +182,21 @@ def replace_observer_with_quantize_dequantize_node(graph: Graph, node: Node, mod
...
             cur_output_node_idx = output_node_seen_cnt
             output_node_seen_cnt += 1
             if cur_output_node_idx in output_quantized_idxs:
                 # Result are kept quantized if the user specified the
                 # output_quantized_idxs override.
-                # TODO: remove dequantize node if any
-                raise Exception("output_quantized_idxs is not supported yet")
+                # Remove the dequantize operator in the end
+                maybe_dequantize_node = node.args[0]
+                if isinstance(maybe_dequantize_node, Node) and maybe_dequantize_node.op == "call_method" and maybe_dequantize_node.target == "dequantize":
```
Comment (vkuzo) [inline torch/ao/quantization/fx/_convert_do_not_use.py:196]: nit: fix lint

**Review by vkuzo — APPROVED** [review]

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32271511).

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32271511).

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32271511).

**Pull request closed without merging**